develop/tutorials/articles/100-tooling/03-blade-cli/05-deploying-projects-with-blade-cli.markdown

```diff
@@ -35,31 +35,42 @@ framework using Felix Gogo shell and deploys the project directly to the OSGi
 container using Felix File Install commands. The command uses the default
 `11311` port by default.
 
-<!--
-You can also specify a custom port to deploy your module to using the `-p`
-parameter followed by the port number. For instance, you could run `blade deploy
--p 8090` to deploy to port 8090.
--->
++$$$
+
+**Note:** The `blade deploy` command requires a Gradle/Maven wrapper to
```

Comment:
We should change 'blade deploy' to just execute a system 'gradle' or 'maven' if the wrapper doesn't exist.  So I think we can remove this statement.

Reply:
@gamerson should I keep this information in the docs until `blade deploy` is changed? I was going to add the deployment changes in a different PR when it was available.